### DISCUSSION TITLE
部分文章底部有滚动条是什么情况？

### DISCUSSION THREAD

**Discussion by katorly** [aiokr/hexo-theme-type#31] [Q&A]
首先Type主题的简约和美观吸引了我，不过
我还没有自己使用这个主题，因为我一般是先挑选好，之后有空再弄。  

在浏览作者自己的博客（来查看样式）的时候，我意外发现生成了目录的文章底部出现了滚动条，而没有目录的文章不会。  
即使我放大或缩小浏览器界面，这个滚动条依然存在。  

例如[这篇文章](https://tripper.press/2022/83d6)和[这篇文章](https://tripper.press/2022/7cce)有目录，我这里看底部就有滚动条。  
而[这一篇](https://tripper.press/2022/8f44)由于没有目录，就没有出现多余的底部滚动条。
![image](https://user-images.githubusercontent.com/70022443/214544059-8466768f-b120-4b80-95a3-9a055e78c2e6.png)

敢问是刻意这样设计的还是只有我浏览会出现这个问题？  
（说实话我不怎么会hexo，看到这样的小问题就有点不敢用的样子🤣）

**Comment by aiokr** [comment]
这个……我这边没有复现出来这个问题唉，可以问一下你的分辨率和浏览器是什么吗

**Reply by katorly** [reply]
![image](https://user-images.githubusercontent.com/70022443/218251242-6d6b0474-09bd-4553-a297-6140eee47456.png)
![image](https://user-images.githubusercontent.com/70022443/218251255-5b50d8b4-3d68-4d9b-947e-f9a0436d6e73.png)

**Comment by aiokr** [comment]
应该是目录或者代码块的问题

**Reply by katorly** [reply]
我现在看这个问题貌似还存在，不过您是不是准备大改样式了？
